src/ontology/reports/uberon-edit.obo-obo-report.tsv

```diff
@@ -4849,6 +4849,7 @@ WARN	missing_definition	UBERON:6007046	IAO:0000115
 WARN	missing_definition	UBERON:6007245	IAO:0000115	
 WARN	missing_definition	UBERON:6007288	IAO:0000115	
 WARN	missing_definition	UBERON:6007289	IAO:0000115	
+WARN	missing_definition	UBERON:8440076	IAO:0000115	
```

Comment:
Lets try to avoid moving forward new terms being added without definitions..

Reply:
hmm thats strange, let me check on that

Reply:
argh thats my bad, I put the def in the wrong field, I'll fix it

Reply:
No need to rerun a release because of this but great :D